api/package.json

```
@@ -3,7 +3,7 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "start": "node ./bin/www"
+    "start": "nodemon ./bin/www"
```

since we use `node ./bin/www` to run the app in server. Let's make this change here:
```suggestion
    "dev": "nodemon ./bin/www",
    "start": "node ./bin/www"
```

Also make sure you update `README` as well and ask developers to use `npm run dev` to start the application locally